From 6200b6bd631403fb7469de39e8af10d2a83a0149 Mon Sep 17 00:00:00 2001
From: NBruderman <nbruderman@gmail.com>
Date: Mon, 6 Apr 2015 19:56:20 +0300
Subject: [PATCH 25/74] FM: fixup strings

* Remove extra space
* Rename "Czech"  to "Czech Republic", as it really named

Change-Id: Iba9dfedd9a4815a2c06ef3bc8b09ce6fd8a38b56
(cherry picked from commit b38c8de3d60cc6bc9ba56fd7a510a15c2ce45c25)
---
 fmapp2/res/values/arrays.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/fmapp2/res/values/arrays.xml b/fmapp2/res/values/arrays.xml
index 05611b8..3ca47df 100644
--- a/fmapp2/res/values/arrays.xml
+++ b/fmapp2/res/values/arrays.xml
@@ -208,7 +208,7 @@
     <item>Belgium</item>
     <item>Brazil</item>
     <item>China</item>
-    <item>Czech</item>
+    <item>Czech Republic</item>
     <item>Denmark</item>
     <item>Finland</item>
     <item>France</item>
@@ -233,7 +233,7 @@
     <item>Sweden</item>
     <item>Taiwan</item>
     <item>Turkey</item>
-    <item>United Kingdom </item>
+    <item>United Kingdom</item>
     <item>United States</item>
     <item>User defined band</item>
     <item>Indonesia</item>
@@ -290,7 +290,7 @@
     <item>Belgium (87.5MHz to 108.0MHz in 100 KHz steps)</item>
     <item>Brazil (87.5MHz to 108.0MHz in 200 KHz steps)</item>
     <item>China (87.0MHz to 108.0MHz in 100 KHz steps)</item>
-    <item>Czech (87.5MHz to 108.0MHz in 100 KHz steps)</item>
+    <item>Czech Republic (87.5MHz to 108.0MHz in 100 KHz steps)</item>
     <item>Denmark (87.5MHz to 108.0MHz in 50 KHz steps)</item>
     <item>Finland (87.5MHz to 108.0MHz in 100 KHz steps)</item>
     <item>France (87.5MHz to 108.0MHz in 100 KHz steps)</item>
-- 
2.17.1

